nats/nats.go

```diff
@@ -0,0 +1,207 @@
+// Package nats abstracts publish-subscribe https://nats.io backend.
+//
+// WARNING/TODO: messages can be considered not acknowledged,
+//               because DurableName for subscriptions is not used yet:
+//               https://godoc.org/github.com/nats-io/stan.go#DurableName
```

Comment:
@dim let's use `client_id` for DurableName? It's to persist Ack-s (smth like kafka group ID).

Reply:
I'll leave the comment though - it's a proper TODO for future.